Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -314,15 +327,22 @@ void MainWidget::initialize_menubar(GUI::Window& window)
     m_paste_action = GUI::CommonActions::make_paste_action([&](auto&) {
         auto* editor = current_image_editor();
         if (!editor) {
-            create_image_from_clipboard();
+            auto result = create_image_from_clipboard();
+            if (result.is_error()) {
+                GUI::MessageBox::show_error(&window, DeprecatedString::formatted("Failed to create image from clipboard: {}", result.error()));
+            }
             return;
         }
 
         auto bitmap = GUI::Clipboard::the().fetch_data_and_type().as_bitmap();
         if (!bitmap)
             return;
 
-        auto layer = PixelPaint::Layer::try_create_with_bitmap(editor->image(), *bitmap, "Pasted layer").release_value_but_fixme_should_propagate_errors();
+        auto layer_result = PixelPaint::Layer::try_create_with_bitmap(editor->image(), *bitmap, "Pasted layer");
+        if (layer_result.is_error()) {
+            GUI::MessageBox::show_error(&window, DeprecatedString::formatted("Could not create bitmap when pasting: {}", layer_result.error()));
```

Comment:
You need to `return;` after showing the error, otherwise we'll try to get the non-existent value out of `layer_result` below and crash.